tests/admin/dead_letter_queue/test_dlq.py

```diff
@@ -2,4 +2,4 @@
 
 
 def test_dlq() -> None:
-    assert len(get_dlq_topics()) == 9
```

Comment:
Feel free to delete this test, IMO that's just annoying to deal with as you add new datasets.